main.rb

```diff
@@ -1,37 +1,100 @@
-require_relative 'person'
-require_relative 'student'
-require_relative 'teacher'
-require_relative 'classroom'
-require_relative 'book'
-require_relative 'rental'
+require_relative './app'
 
-# Creating a Classroom
-classroom = Classroom.new('Class A')
+def create_student(app)
+  print 'Age: '
+  age = gets.chomp
+  print 'Name: '
+  name = gets.chomp
+  print 'Has parent permission? [Y/N]: '
+  permission = gets.chomp.downcase == 'y'
+  app.create_student(age, name, permission)
+  puts 'Person created successfully'
+end
 
-# Creating students and assigning them to the classroom
-Student.new(1, classroom, name: 'John', age: 16)
-Student.new(2, classroom, name: 'Jane', age: 17)
+def create_teacher(app)
+  print 'Age: '
+  age = gets.chomp
+  print 'Name: '
+  name = gets.chomp
+  print 'Specialization: '
+  specialization = gets.chomp
+  app.create_teacher(age, name, specialization)
+  puts 'Person created successfully'
+end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md#:~:text=in%20the%20example.-,Create%20a%20app.rb%20file%20that%20will%20serve%20as%20your%20console%20app%20entry%2Dpoint.%20It%20should%20have%20methods%20that%20do%20the%20following%3A,-List%20all%20books), Please move these methods (create_student, create_teacher, create_person, create_rentals, create_book, and list_rentals) to the `app.rb` module and only leave `show_options` and `main` methods here in `main.rb` module because this module as pre this r[equirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md#:~:text=given%20person%20id.-,In%20your%20main.rb%20define%20the%20entry%20point%2C%20this%20will%20be%20a%20method%20called%20main%20that%20is%20invoked%20at%20the%20end%20of%20your%20file.%20This%20method%20should%20do%20the%20following%3A,-Present%20the%20user) is only supposed to do the following:
- Present the user with a list of options to perform
- Lets users choose an option.
- Have a way to quit the app.